### ISSUE TITLE
getScrollbarWidth shouldn't use unsafe styles

### ISSUE THREAD

**Issue by kiliancs** [xobotyi/react-scrollbars-custom#469]
**What is the current behavior?**

With a CSP of `style-src 'self'`, the following error or similar is logged by the browser:

```
Content-Security-Policy: The page’s settings blocked the loading of a resource at inline (“default-src”).
```

The cause is this code in `getScrollbarWidth` in `src/util.tsx`:

```tsx
  el.setAttribute(
    'style',
    'position:absolute;width:100px;height:100px;top:-999px;left:-999px;overflow:scroll;'
  );
```

The content inside the `Scrollbar` component is not visible, and scrolling is not possible.

**Steps to reproduce it and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code and it doesn't have extra dependencies other than `react-scrollbars-custom`. Paste the link to your [JSFiddle](https://jsfiddle.net) or [CodeSandbox](https://codesandbox.io) example below:**

https://codesandbox.io/s/autumn-fast-n5p48h?file=/public/index.html

Basically, add the following meta tag to the HTML of your project:

```html
    <meta
      http-equiv="Content-Security-Policy"
      content="style-src 'self'"
    />
```

**What is the expected behavior?**

No CSP violations from react-scrollbars-custom. Scrolling works.

**A little about versions:**  
- _OS_: Ventura 13.3.1
- _Browser (vendor and version)_: Firefox 118.0.1
- _React_: 17.0.1
- _`react-scrollbars-custom`_: 4.1.1
- _Did this worked in the previous package version?_: Not sure but I wouldn't expect so.
